vtStor.ATest/main.cpp

```diff
@@ -40,8 +48,13 @@ void main()
     vtStorInit( driveManager );
 
     vtStor::cAta::s_DefaultCommandHandlerCommandType = 0;
+    vtStor::cScsi::s_DefaultCommandHandlerCommandType = 0;
```

Comment:
These were only needed when we had the enumerator created command handler and protocol.
Let's make note and remove the to avoid confusing, for the next time.